app/src/main/java/com/x8bit/bitwarden/ui/platform/components/card/BitwardenActionCard.kt

```diff
@@ -0,0 +1,121 @@
+package com.x8bit.bitwarden.ui.platform.components.card
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.BoxScope
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.VectorPainter
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.x8bit.bitwarden.R
+import com.x8bit.bitwarden.ui.platform.components.util.rememberVectorPainter
+import com.x8bit.bitwarden.ui.platform.theme.BitwardenTheme
+
+/**
+ * A reusable card for displaying actions to the user.
+ */
+@Composable
+fun BitwardenActionCard(
+    actionIcon: VectorPainter,
+    actionText: String,
+    callToActionText: String,
+    onCardClicked: () -> Unit,
+    modifier: Modifier = Modifier,
+    trailingContent: (@Composable BoxScope.() -> Unit)? = null,
+) {
+    Card(
+        onClick = onCardClicked,
+        shape = RoundedCornerShape(size = 16.dp),
+        modifier = modifier
+            .fillMaxWidth()
+            .wrapContentHeight(),
```

Comment:
I would expect layout modifiers to be defined where the `BitwardenActionCard` is used to ensure this can be reused.